tests/lxd/test_lxd_backend.cpp

```diff
@@ -1897,3 +1898,38 @@ TEST_F(LXDBackend, posts_network_data_config_if_available)
     mp::LXDVirtualMachine machine{default_description, stub_monitor, mock_network_access_manager.get(), base_url,
                                   bridge_name};
 }
+
+namespace
+{
+class CustomLXDFactory : public mp::LXDVirtualMachineFactory
+{
+public:
+    using mp::LXDVirtualMachineFactory::create_bridge_with;
+    using mp::LXDVirtualMachineFactory::LXDVirtualMachineFactory;
+
+    MOCK_METHOD2(prepare_networking_guts,
+                 void(std::vector<mp::NetworkInterface>& extra_interfaces, const std::string& bridge_type));
+};
+} // namespace
+
+TEST_F(LXDBackend, prepares_networking_via_base_factory)
+{
+    CustomLXDFactory backend{std::move(mock_network_access_manager), data_dir.path(), base_url};
+    std::vector<mp::NetworkInterface> extra_networks{{"netid", "mac", false}};
+
+    auto address = [](const auto& v) { return &v; }; // replace with Address matcher once available
+    EXPECT_CALL(backend, prepare_networking_guts(ResultOf(address, Eq(&extra_networks)), Eq("bridge")));
+    backend.prepare_networking(extra_networks);
+}
+
+TEST_F(LXDBackend, createsBridgesViaBackendUtils)
```

Comment:
```suggestion
TEST_F(LXDBackend, creates_bridges_via_backend_utils)
```

A bunch more below that are camelCase.

Reply:
I asked the same thing to @townsend2010 recently and he reminded me that, as I had pointed out to him, [gtest advises against that](https://github.com/google/googletest/blob/master/docs/faq.md#why-should-test-suite-names-and-test-names-not-contain-underscore) :slightly_smiling_face: So he's been switching to camelCase recently and I thought it only made sense to start doing the same... WDYT?

Reply:
+1 on camelCase, then, as long as we try and be consistent...